File: Folder Java/TokoElektronik.java
class TokoElektronik {
    // Data produk
    private String namaProduk = "-";
    private String merk = "-";
    private String noSeri = "-";
    private String deskripsi = "-";
    private String harga = "-";
    private int stok = 0;

    // --- Getter & Setter ---
    public String getNamaProduk() {
        return namaProduk;
    }

    public void setNamaProduk(String namaProduk) {
        this.namaProduk = namaProduk;
    }

    public String getMerk() {
        return merk;
    }

    public void setMerk(String merk) {
        this.merk = merk;
    }

    public String getNoSeri() {
        return noSeri;
    }

    public void setNoSeri(String noSeri) {
        this.noSeri = noSeri;
    }

    public String getDeskripsi() {
        return deskripsi;
    }

    public void setDeskripsi(String deskripsi) {
        this.deskripsi = deskripsi;
    }

    public String getHarga() {
        return harga;
    }

    public void setHarga(String harga) {
        this.harga = harga;
    }

    public int getStok() {
        return stok;
    }

    public void setStok(int stok) {
        this.stok = stok;
    }

    // --- Method tampil data produk ---
    public void viewProduk() {
        System.out.println("Nama Produk   : " + namaProduk);
        System.out.println("Merk          : " + merk);
        System.out.println("No Seri       : " + noSeri);
        System.out.println("Deskripsi     : " + deskripsi);
        System.out.println("Harga         : Rp " + harga);
        System.out.println("Stok          : " + stok);
        System.out.println();
    }
}

File: Folder Python/TokoElektronik.py
class TokoElektronik:
    def __init__(self):
        # Hanya data produk
        self.namaProduk = "-"
        self.merk = "-"
        self.noSeri = "-"
        self.deskripsi = "-"
        self.harga = "-"
        self.stok = 0

    # Getter & Setter Produk
    def setNamaProduk(self, nama):
        self.namaProduk = nama
    def getNamaProduk(self):
        return self.namaProduk

    def setMerk(self, merk):
        self.merk = merk
    def getMerk(self):
        return self.merk

    def setNoSeri(self, noSeri):
        self.noSeri = noSeri
    def getNoSeri(self):
        return self.noSeri

    def setDeskripsi(self, deskripsi):
        self.deskripsi = deskripsi
    def getDeskripsi(self):
        return self.deskripsi

    def setHarga(self, harga):
        self.harga = harga
    def getHarga(self):
        return self.harga

    def setStok(self, stok):
        self.stok = stok
    def getStok(self):
        return self.stok

    # View Produk
    def viewProduk(self):
        print(f"Nama Produk   : {self.namaProduk}")
        print(f"Merk          : {self.merk}")
        print(f"No Seri       : {self.noSeri}")
        print(f"Deskripsi     : {self.deskripsi}")
        print(f"Harga         : Rp {self.harga}")
        print(f"Stok          : {self.stok}")
        print("")
